zookeeper-server/src/test/java/org/apache/zookeeper/test/ThrottledOpQuorumTest.java

```diff
@@ -23,15 +23,22 @@
 import org.apache.zookeeper.ZooKeeper;
 import org.apache.zookeeper.server.ZooKeeperServer;
 import org.apache.zookeeper.server.quorum.QuorumPeer;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 public class ThrottledOpQuorumTest extends QuorumBase {
-    @BeforeClass
+    @BeforeAll
     public static void applyMockUps() {
         ThrottledOpHelper.applyMockUps();
     }
 
+    @BeforeEach
+    @Override
+    public void setUp() throws Exception {
```

Comment:
why do we need this method? (maybe I just miss something)

Reply:
Fixed

Reply:
I still see this method here. Did you fix this in the parent class somehow? I still see a `setUp()` method in the `QuorumBase` class with `@BeforeEach` annotation, so this method doesn't have any benefit (unleess if I miss something).

Reply:
This time it should be really fixed. Sorry, I haven't recognized that I forgot them.